nupic/research/monitor_mixin/plot.py

```diff
@@ -22,9 +22,13 @@
 """
 Plot class used in monitor mixin framework.
 """
-import matplotlib.pyplot as plt
-import matplotlib.cm as colorModel
+import traceback
 
+try:
+  # We import in here to avoid creating a matplotlib dependency in nupic.
+  import matplotlib.pyplot as plt
```

Comment:
I've moved the import to this try/catch, deleted the import from the method, and the build passes.